### PR TITLE
Query and update

### PR DESCRIPTION
**Pull request by lucascomamala**
- Use database transactions.
- Modify and delete data in SQL.
- Prepare complex queries that answer analytical questions.

Screenshot of Transactions:
197883499-5cc7d88e-bed7-48e2-8c43-58a82d59afa2

Screenshot of Queries:
197897474-d0e091d7-c0be-4ab0-be61-a49bab3ed753

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -8,3 +9,32 @@ SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
 SELECT * from animals WHERE neutered IS true;
 SELECT * from animals WHERE name NOT IN ('Gabumon');
 SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+/* Day 2 */
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
```
Comment (Deepakdanger) [inline queries.sql:16]: - [ ] Kindly use 
 BEGIN; -- start transaction

   UPDATE animals
   SET species = 'unspecified'; 

   SELECT species from animals; 
   ROLLBACK;

   SELECT species from animals;

File: queries.sql [hunk 2]
```diff
@@ -8,3 +9,32 @@ SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
...
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS null;
+COMMIT;
```
Comment (Deepakdanger) [inline queries.sql:21]: - [ ] Kindly commit the transaction.

```sql
   BEGIN;
   UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon'; 
   UPDATE animals SET species = 'pokemon' WHERE species IS NULL; 

   SELECT species from animals; 
   COMMIT;
   SELECT species from animals;
```

File: queries.sql [hunk 3]
```diff
@@ -8,3 +9,32 @@ SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
...
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
```
Comment (Deepakdanger) [inline queries.sql:25]: - [ ] Inside a transaction delete all records in the animals table, then roll back the transaction.

```sql
   BEGIN;

   DELETE FROM animals WHERE ;
   SELECT COUNT(*) FROM ANIMALS;

   ROLLBACK;
   SELECT COUNT(*) FROM ANIMALS;
```

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @lucascomamala , 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — APPROVED** [review]
Hi @lucascomamala , 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- NA

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**